packages/lesswrong/lib/collections/automatedContentEvaluations/newSchema.ts

```diff
@@ -34,12 +34,46 @@ const schema = {
       typescriptType: "{ sentence: string; score: number; }[]",
     },
     graphql: {
-      outputType: "[SentenceScore]",
-      canRead: ["admins"],
+      outputType: "[SentenceScore]!",
```

Comment:
graphql has the unfortunate property that this translates to `Array<SentenceScore | null>` in practice; it needs to be `[SentenceScore!]!` if you want to forbid null values _inside_ the array (and not just forbid top-level null values)